mii/models/score/score_template.py

```diff
@@ -16,6 +16,8 @@
 
 def init():
     model_path = mii.utils.full_model_path(configs[mii.constants.MODEL_PATH_KEY])
+    deployment_tag = configs[mii.constants.DEPLOYMENT_TAG_KEY]
+    deployments = mii.multi_model_deployments[deployment_tag]
```

Comment:
I don't think this will work. I believe the `mii.multi_model_deployments` that you add to in `deployment.py` is on a different process than the one that will be running the generated `score.py` file. So this will just be an empty dict when the score file gets executed!

Reply:
Oh, ok that makes sense!